tn_api_hotel.yml

```diff
@@ -239,89 +312,449 @@ components:
+                        type: string
+                        example: 2G3C
+                    provider:
+                        description: Fare provider to use.<br  />“1V” - Apollo, “1G” - Galileo, “1P” - Worldspan, “1A” - Amadeus
+                        required: true
+                        type: string
+                        example: 1V
+                    pcc_currency:
+                        description: The currency that is associated with the PCC. This will be the currency that any bookings are made and charged in.
+                        type: string
+                        example: CAD
+            hotel_chain:
+                type: string
+                description: The refence for the hotel chain.
+                example: 'GI'
+            hotel_code:
+                type: string
+                description: The reference for the hotel code
+                example: '35573'
+            total_travellers:
+                type: int
+                example: 3
+                description: Number of people that will be staying at the hotel.
+            total_rooms:
+                type: int
+                example: 2
+                description: The number of rooms required for the above travellers.
+            currency:
+                type: string
+                example: 'CAD'
+                description: The currency that the details should be presented in.
+            check_in_date:
+                type: string
+                format: date
+                description: Date of check-in to search hotel availibilities on.
+            check_out_date:
+                type: string
+                format: date
+                description: Date to checkout from the hotel.
+
+    HotelDetailsResponse:
+        properties:
+            super_trip_id:
+                type: string
+                example: "bdf876sf6sfsdf87sdf"
+                description: id of the super trip
+            hotel_id:
                 type: string
-                description: An identifier to book the room with
                 example: 'ndsf6sjf6sdf7sdf6'
-              hotel_name:
+                description: An identifier to refer to the hotel for searching room availability
+            address:
+                type: object
+                properties:
+                    street_address:
+                        type: string
+                        description: The street address of the hotel.
+                        example: '1583 Brunswick St, Halifax, NS B3J 3P5'
+                    longitude:
+                        type: float
+                        description: The longitude of the hotel.
+                        example: 44.64511447836071
+                    lattitude:
+                        type: float
+                        description: The lattitude of the hotel.
+                        example: -63.575245615440565
+                    distance_from_city_km:
```

Comment:
Same comment as in the availability piece - the distance provided by Travelport is from the reference point (airport for YHZ)